clouddrift/sphere.py

```diff
@@ -43,9 +43,6 @@ def recast_lon(lon: np.ndarray, lon0: Optional[float] = -180) -> np.ndarray:
     --------
     :func:`recast_lon360`, :func:`recast_lon180`
     """
-    if np.isscalar(lon):
-        lon = np.array([lon])
-
```

Comment:
@philippemiron do you know why we haven't been accepting a scalar in `recast_lon`? When testing `haversine.position_from_distance_and_bearing`, I came to want to pass a scalar to this. All the sphere tests pass with the change in this commit, and I can't tell that any aspect of the function changed for the worse. Is there some gotcha here that I don't see?